bartop-ui/src/components/AuthButton/AuthButton.js

```diff
@@ -3,12 +3,19 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
 import Button from '../Button/Button';
-import { actions } from '../../ducks/authentication/authentication';
+import {
+  actions as modalActions,
+  MODAL_TYPES
+} from '../../ducks/modals/modals';
+import { actions as authActions } from '../../ducks/authentication/authentication';
 import strings from '../../strings';
 
-export const AuthButton = ({ className, loggedIn, login, logout }) =>
+export const AuthButton = ({ className, loggedIn, showModal, logout }) =>
   !loggedIn ? (
-    <Button className={className} onClick={login}>
+    <Button
+      className={className}
+      onClick={() => showModal(MODAL_TYPES.LOGIN_MODAL)}
+    >
```

Comment:
I mentioned this on Slack but I have a UX question - do you think we should make the button more descriptive to handle logging in but also signing up?

Reply:
Yeah fo sho, I'll open a separate issue for thinking about button wording / login vs. sign up

Reply:
https://app.zenhub.com/workspace/o/bartop-io/bartop/issues/191